src/components/pages/result_page.py

```
@@ -1,23 +1,100 @@
+    with st.spinner("プレビューを準備中..."):
+        if selected_format == "PDF":
+            # 既存のPDFデータをプレビュー用に使用
+            preview_data = file_data
+        else:
+            # HTML/PPTXは一度PDFに変換してからプレビュー
+            preview_data = converter.convert_template_to_pdf(template)
+        images = convert_from_bytes(preview_data)
+    for i, image in enumerate(images):
+        st.image(image, caption=f"スライド {i+1}")
+
+except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)